streamingpro-mlsql/src/main/java/streaming/core/strategy/platform/SQLTableCacheClean.scala

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Make sure you get all sessions from SessionManager, and `SparkSessionCacheManager` will show you an example of how to close sessions.